src/nav/infrastructure.yml

```diff
@@ -454,8 +454,10 @@ pages:
         pages:
           - title: ALB/NLB integration
             path: /docs/infrastructure/amazon-integrations/aws-integrations-list/aws-albnlb-monitoring-integration
-          - title: API Gateway integration
-            path: /docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-monitoring-integration
+          - title: API Gateway V1 integration
```

Comment:
```suggestion
          - title: API Gateway v1 integration
```

Reply:
@rhetoric101, I made the suggested change.